docs/how-to/namespace-transition.md

```diff
@@ -46,41 +75,84 @@ release from the `knative-gcp` repository. We will update the page with specific
    kubectl delete service controller -n cloud-run-events
    ```
 
-   For your convenience, we also packaged these instructions as a shell script
-   and as a yaml file that you can execute as shown below.
-
-   ```shell script
-   # execute the setup commands using the provided script
-   bash namespace-transition.sh
-   ```
+#### Using Workload Identity
 
-   Or, if you prefer executing these commands as a job, apply the following yaml
-   file.
+1. For this purpose we need to first setup a number of environment variables
+   based on the existing authentication settings. The following `gcloud` command
+   can be used to view the existing Google Service Accounts (GSAs) and their
+   roles in your GCP project.
 
    ```shell script
-   # execute the setup commands as a job
-   kubectl apply -f namespace-transition.yaml
+   export PROJECT_ID=<gcp-project-identifier>
```

Comment:
Which `<gcp-project-identifier>` should I pick? If I have multiple projects, is this the project the cluster is in?

Reply:
Added a comment before defining the variable.